action.yml

```diff
@@ -153,4 +153,7 @@ runs:
           diff -B west.yml tmp.yml | patch west.yml - || true
           git add west.yml
           git commit --amend --no-edit
+          git fetch origin
+          git merge-tree --write-tree --name-only origin/main auto-manifest-${{ github.event.repository.name }}-${{ github.event.pull_request.number }}
```

Comment:
checkout step has git cloned the forked repo so I would assume origin/main points to main of forked repo (which is not updating automatically). So I assume you would need to add git remote which points to actual sdk-nrf.